app/helpers/pages_helper.rb

```diff
@@ -24,6 +24,19 @@ def ak_resource_id(ak_resource_url)
     match[1]
   end
 
+  def ak_page_type(ak_resource_url)
+    match = %r{\/(\w+)\/(\d+)\/?$}.match(ak_resource_url)
```

Comment:
Could we put a short comment above this with an example input and what we're trying to match/output? it's difficult to tell otherwise

Reply:
Done!